code/parachain/runtime/composable/src/governance.rs

```diff
@@ -80,10 +80,16 @@ impl pallet_referenda::Config for Runtime {
 		system::EnsureSignedBy<TechnicalCommitteeMembership, Self::AccountId>,
 		system::EnsureSignedBy<CouncilMembership, Self::AccountId>,
 	>;
-	
+
+	#[cfg(not(feature = "fastnet"))]
 	type CancelOrigin = EnsureRootOrOneThirdNativeTechnical;
+	#[cfg(feature = "fastnet")]
+	type CancelOrigin = EnsureRootOrOneSixthNativeTechnical;
 
+	#[cfg(not(feature = "fastnet"))]
 	type KillOrigin = EnsureRootOrMoreThenHalfNativeCouncil;
+	#[cfg(feature = "fastnet")]
```

Comment:
this is testnet - security. but okey like that. good.

time = fastnet
amount of signatrues = testnet